platform/views/platformViews.groovy

```diff
@@ -46,11 +46,11 @@ branchList.each { branch ->
 
 }
 
-listView("edx-platform-pipeline-pr-tests") {
+listView("edx-platform-pr-tests") {
```

Comment:
A view of this name already exists, created by iteration over `branchList` above.  Do we need to keep this separate one anymore?

Reply:
Good point. Removed. No need to separate anymore